07-sampling.Rmd

```diff
@@ -594,637 +532,936 @@ if (is_latex_output()) {
 }
 ```
 
-Observe that as the sample size increases, the variation of the 1000 replicates of the proportion of red decreases. In other words, as the sample size increases, there are fewer differences due to sampling variation and the distribution centers more tightly around the same value. Eyeballing Figure \@ref(fig:comparing-sampling-distributions), all three histograms appear to center around roughly 40%.
 
-We can be numerically explicit about the amount of variation in our three sets of 1000 values of `prop_red` using the \index{standard deviation} *standard deviation*. A standard deviation is a summary statistic that measures the amount of variation within a numerical variable (see Appendix \@ref(appendix-stat-terms) for a brief discussion on the properties of the standard deviation). For all three sample sizes, let's compute the standard deviation of the 1000 proportions red by running the following data wrangling code that uses the `sd()` summary function.
+Observe that all three histograms are:
 
-```{r, eval=FALSE}
-# n = 25
-virtual_prop_red_25 %>% 
-  summarize(sd = sd(prop_red))
+- centered around the same middle value, which appears to be a value slightly below 0.4,
+- are somewhat bell-shaped, and
+- exhibit *sampling variation* that is different for each sample size. In particular, as the sample size increases from 25 to 50 to 100, the sample proportions do not vary as much and they seem to get closer to the middle value.
 
-# n = 50
-virtual_prop_red_50 %>% 
-  summarize(sd = sd(prop_red))
+These are important characteristic of the *sampling distribution* of the sample proportion: the first observation relates to the shape of the distribution, the second to the center of the distribution, and the last one to the *sampling variation* and how it is affected by the sample size. These results are not coincidental or isolated to the example of sample proportions of red balls in a bowl. In the next subsection, a theoretical framework is introduced that helps explain with precise mathematical equations the behavior of sample proportions obtained from random samples.
 
-# n = 100
-virtual_prop_red_100 %>% 
-  summarize(sd = sd(prop_red))
+
+<!-- 
+Learning checks need to be updated. AV 8/17/23
+-->
+
+
+```{block, type="learncheck", purl=FALSE}
+\vspace{-0.15in}
+**_Learning check_**
+\vspace{-0.1in}
 ```
 
-Let's compare these three measures of distributional variation in Table \@ref(tab:comparing-n).
+**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** As shown in Figure \@ref(fig:comparing-sampling-distributions) the histograms of sample proportions are somewhat bell-shaped. What can you say about the center of the histograms?
 
-```{r comparing-n, echo=FALSE, purl=FALSE}
-comparing_n_table <- virtual_prop %>%
-  group_by(n) %>%
-  summarize(sd = sd(prop_red)) %>%
-  rename(`Number of slots in shovel` = n, `Standard deviation of proportions red` = sd)
+- A. The larger the sample size the greater the center of the histogram.
```

Comment:
Maybe "more concentrated the center" or "more peaked vs flatter the center"? Greater the center might confuse some people into thinking greater values on the x-axis